apps/grader-host/src/index.ts

```diff
@@ -169,6 +180,10 @@ async function isJobCanceled(job: GradingJobMessage) {
   return result.rows[0].canceled;
 }
 
+function run<T>(fn: () => T): T {
+  return fn();
+}
```

Comment:
https://maxgreenwald.me/blog/do-more-with-run

How do folks feel about this? IMO the ergonomics of this little thing are really nice compared to some of the alternatives.

Reply:
I love it ❤️. Especially for use in `html` templates to avoid nesting ternary operators 🤮